cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -178,3 +178,9 @@ def test_delete_instance(new_instance):
         assert False, "entity still exists"
     except ItemNotFound:
         pass
+
+
+def test_instance_operating_system_linux(new_instance):
+    view = navigate_to(new_instance, 'Details')
+    os = view.entities.summary('Properties').get_text_of("Operating System")
+    assert os == new_instance.provider.data['provisioning']['image']['os_distro']
```

Comment:
Required: add a message in case assert fails

Reply:
Done